packages/core/src/mpi/filter-patients/match-business-rules.ts

```diff
@@ -0,0 +1,149 @@
+import { buildDayjs } from "@metriport/shared/common/date";
+import { PatientData } from "../../domain/patient";
+import { calculateLastNameScore } from "./match-name";
+
+export function checkBusinessRules(
```

Comment:
make the code self doucmenting - give a lot of the conditions a name and then use those in your if conditions.

const noNameMatch = scores.names === 0
const noAddressMatch = scores.address < 2
const lowDobMatch = scores.dob <= 1
const noDobMatch = scores.dob === 0
const lastNameScore = calculateLastNameScore(metriportPatient, externalPatient)
...